NuGet.config

```diff
@@ -6,6 +6,11 @@
   </fallbackPackageFolders>
   <packageSources>
     <clear />
+    <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <!--  Begin: Package sources from dotnet-corefx -->
+    <add key="darc-int-dotnet-corefx-59d2f36" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-corefx-59d2f36e/nuget/v3/index.json" />
+    <!--  End: Package sources from dotnet-corefx -->
+    <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
```

Comment:
we will need to remove all the darc-int feeds

Reply:
is it enough to revert this file?

Reply:
yes we can revert the changes as well